changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -1,5 +1,23 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "4D251ED7-CD9A-431E-8FB5-BA2DD49E3B35",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "accessReviewError",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **accessReviewError** property to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstance?view=graph-rest-beta) resource.",
+                    "Target": "accessReviewInstance"
```

Also add an entry for the accessReviewError resource.
```suggestion
                    "Id": "4D251ED7-CD9A-431E-8FB5-BA2DD49E3B35",
                    "ApiChange": "Property",
                    "ChangedApiName": "errors",
                    "ChangeType": "Addition",
                    "Description": "Added the **errors** property to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstance?view=graph-rest-beta) resource.",
                    "Target": "accessReviewInstance"
```